GDAXSharp/WebSocket/Models/Request/TickerChannel.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace GDAXSharp.WebSocket.Models.Request
+{
+    public class TickerChannel
+    {
+        [JsonProperty("type")]
```

Comment:
shouldn't need all of these attributes

Reply:
Isn't this necessary to build/serialize to JSON?

Reply:
@Sotam - have a look in `AbstractService `at `protected static string SerializeObject` and the deserialize method... they both use specific `JsonSerializerSettings` which take case of the gdax naming convention automatically (in vast majority of cases)

The 2 static methods:  `SerializeObject `/ `DeserializeObject `and static property `SerializerSettings ` could be removed from  AbstractService to a static helper class... then have WebSocket and AbstractService both make use of new helper class?